bridges/scalajs/src/main/scala/bloop/scalajs/JsBridge.scala

```diff
@@ -38,40 +40,31 @@ object JsBridge {
   private def findIrFiles(path: Path): List[Path] =
     Files.walk(path).iterator().asScala.filter(isIrFile).toList
 
-  private def toIRJar(jar: Path) =
+  private def toIrJar(jar: Path) =
     IRContainer.Jar(new FileVirtualBinaryFile(jar.toFile) with VirtualJarFile)
 
   def link(
       config: JsConfig,
       project: Project,
       mainClass: String,
+      target: File,
       logger: BloopLogger
-  ): Path = {
+  ): Unit = {
     val classpath = project.classpath.map(_.underlying)
     val classpathIrFiles = classpath
       .filter(Files.isDirectory(_))
       .flatMap(findIrFiles)
       .map(f => new FileVirtualScalaJSIRFile(f.toFile))
-
-    val outputPath = project.out.underlying
-    val target = project.out.resolve("out.js")
-
-    val enableOptimizer = config.mode match {
-      case LinkerMode.Debug => false
-      case LinkerMode.Release => true
-    }
-
-    val jarFiles = classpath.filter(isJarFile).map(toIRJar)
+    val optimizer = config.mode == LinkerMode.Release
+    val jarFiles = classpath.filter(isJarFile).map(toIrJar)
```

Comment:
I think this is unused?

Reply:
Do you mean `jarFiles`? It is accessed in the line below.

Reply:
Ugh sorry